README.md

```diff
@@ -250,6 +250,16 @@ the [Lombok Gradle Plugin](https://plugins.gradle.org/plugin/io.freefair.lombok)
 to delombok your source code before it is passed to the Checker Framework
 for typechecking. This plugin does not support any other use of Lombok.
 
+## Incompatibility with explicitly-specified annotation processors
+
+If you provide a `-processor` option to `javac`,
+this plugin will have no effect, because it relies on
+`javac`'s annotation processor discovery mechanism. The
+`-processor` flag disables the discovery mechanism. You should
```

Comment:
Move this sentence after "have no effect".  Put the explanation of what to do first, and put it together rather than breaking it up with explanation or background.